triton-profiler/src/triton_profiler.rs

```diff
@@ -219,10 +225,14 @@ impl TritonProfiler {
     }
 
     fn plain_stop(&mut self) {
-        let index = self.stack.pop().unwrap().0;
+        let (index, name) = self.stack.pop().unwrap();
         let now = self.timer.elapsed();
         let duration = now - self.profile[index].time;
         self.profile[index].time = duration;
+
+        if std::env::var(GET_PROFILE_OUTPUT_AS_YOU_GO_ENV_VAR_NAME).is_ok() {
+            println!("stop: {name}");
```

Comment:
How about also adding the time it took to run the thing, even without pretty formatting in place?
```suggestion
            println!("stop: {name} (took {duration:.2?})");
```